doc/reference/autoinstall-reference.rst

```diff
@@ -133,79 +116,82 @@ The channel to check for updates.
 .. _ai-keyboard:
 
 keyboard
---------
+~~~~~~~~
 
 * **type:** mapping, see below
 * **default:** US English keyboard
 * **can be interactive:** true
 
-The layout of any attached keyboard. Often systems being automatically
-installed will not have a keyboard at all in which case the value used here
-does not matter.
+The layout of any attached keyboard. Often systems being automatically installed do have a keyboard. In such cases, the setting is not used.
```

Comment:
The meaning has been inverted here.  Maybe we just drop all the text starting with "Often systems  ..."

Reply:
I just forgot the "not" :)

Reply:
> Maybe we just drop all the text starting with "Often systems ..."

True. Redundant.